docs/api/locale.rst

```diff
@@ -0,0 +1,310 @@
+Locale API
```

Comment:
The API docs have a different convention, they are all named in plural -> http://docs.sylius.org/en/latest/api/index.html

That is why it should not be changed.